jobbergate-api/jobbergate_api/apps/applications/routers.py

```diff
@@ -155,15 +149,17 @@ async def applications_delete_upload(
     "/applications/{application_id}",
     status_code=status.HTTP_204_NO_CONTENT,
     description="Endpoint to delete application",
-    dependencies=[Depends(guard.lockdown(Permissions.APPLICATIONS_EDIT))],
 )
 async def application_delete(
     application_id: int = Query(..., description="id of the application to delete"),
+    token_payload: TokenPayload = Depends(guard.lockdown(Permissions.APPLICATIONS_EDIT)),
 ):
     """
     Delete application from the database and S3 given it's id.
     """
-    logger.debug(f"Unlinking job_scripts with links to application {application_id=}")
+    application = await _fetch_and_verify_ownership(application_id, token_payload)
+
+    logger.debug(f"Unlinking job_scripts with links to application {application.id=}")
```

Comment:
are you going to rebase before merging? I believe you changed this line in other PR.

Reply:
I suppose I will have to if there is a conflict here. Need to get some of these PRs merged in before we have too much of a merge nightmare here.